modules/odc-creating-apiserversource.adoc

```diff
@@ -7,9 +7,16 @@
 
 .Procedure
 
-. Navigate to the **Add** page and select **Event Source**.
-. In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-. Configure the **ApiServerSource** settings:
+. In the **Developer** perspective, navigate to **+Add** → **Event Source**. The  **Event Sources** page in the **Developer Catalog** is displayed.
++
+If you have multiple providers, you can filter the available event sources based on the **Provider**.
+. In the **Event Sources** page, select **ApiServerSource** and click **Create Event Source**. The  **Create Event Source** page is displayed.
```

Comment:
Can remove - In the **Event Sources** page, since you mention on line 10 , that it is displayed.
Select **ApiServerSource** and then click

Reply:
Done.